docs/docs/sourcing-from-graphcms.md

```diff
@@ -4,158 +4,238 @@ title: Sourcing from GraphCMS
 
 ## Headless the GraphCMS way
 
-[GraphCMS](https://graphcms.com?ref="gatsby-headless-docs-top") is a headless CMS that is optimized for working with GraphQL. Content structures like posts, authors, products and more are broken down into content types called "models." These models can then be queried with the familiar GraphQL syntax.
-
-One of the benefits of GraphCMS when working with Gatsby is that it supports GraphQL natively which allows you to test your content queries before even starting your Gatsby project.
+[GraphCMS](https://graphcms.com?referrer=gatsby-headless-docs-top) is a GraphQL native Headless Content Management System (Headless CMS) that lets you programmatically deliver content across platforms. With features like an intuitive schema builder, GraphQL mutations API, and out of the box localization, GraphCMS enables you to rapidly build digital projects with your preferred frameworks and languages.
 
 ## Getting started
 
-In this guide you'll create a complete project capable of querying data from GraphCMS.
+In this guide you'll create a Gatsby site capable of querying data from GraphCMS.
+
+### Prerequisites
+
+This guide assumes the following:
 
-### Install the boilerplate
+- You have an active GraphCMS account
+- You've created a new GraphCMS project (preferably using the `Blog Starter` template)
 
-To begin, let's create a Gatsby starter site.
+All schema and data references in this guide are from the `Blog Starter` template.
+
+### Create a new Gatsby site
+
+To begin, let's create a new Gatsby site using the default starter.
 
 ```shell
 gatsby new gatsby-site https://github.com/gatsbyjs/gatsby-starter-default
 ```
 
-Navigate inside of the project with `cd gatsby-site`.
+Once finished, navigate inside of the project with `cd gatsby-site`.
 
-### Add the source plugin
+### Add the `gatsby-source-graphcms` plugin
 
-Additionally, you need the `gatsby-source-graphql` library. Because GraphCMS uses GraphQL natively, you will take advantage of Gatsby's ability to simply stitch two GraphQL APIs together, reducing the time required to transform content. There is no need to use a special gatsby-source-x-cms plugin, the GraphQL source plugin is all you need.
+In order to fetch data from your GraphCMS project, you will need to install [`gatsby-source-graphcms`](https://github.com/GraphCMS/gatsby-source-graphcms).
 
-You can install this component with:
+You can install this package with:
 
 ```shell
-  # Optionally with `npm install`
-  npm install gatsby-source-graphql
+  yarn add gatsby-source-graphcms@next
```

Comment:
I think we need some note about why `@next` here.

Reply:
was not the rule to use in docs the `npm install` and not `yarn`?